src/Elcodi/Component/Tax/Entity/Traits/TaxTrait.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: marcel
+ * Date: 21/05/15
+ * Time: 10:01
+ */
+
+namespace Elcodi\Component\Tax\Entity\Traits;
+
+/**
+ * Trait for entities that need to calculate taxes.
+ *
+ * Product, Variant, CartLine and OrderLine Cart and Orders entities usually will have this trait.
+ *
+ */
+trait TaxTrait
+{
+    public function CalculateTaxAmount($price, $taxPercentage)
+    {
+            return (integer) round( ( $price * $taxPercentage/100 ) );
```

Comment:
Beware of this! we should be using Banker rounding (php `round` with `PHP_ROUND_HALF_ODD`, see http://php.net/manual/es/function.round.php)

We really should ***NOT** do any rounding except for in `CurrencyConverter`